### ISSUE TITLE
[false-positive] `for` loop is marked as error when iteration expression is `*=`

### ISSUE THREAD

**Issue by zerkms** [kalessil/phpinspectionsea#873]
In the following code the line with `for` statement is marked with `Suspicious loop` inspection

```php
<?php

for ($i = 1; $i <= 10; $i *= 2) {}
```
If the iteration expression is changed to `$i++` the inspection goes away.

I probably am missing something, but I don't see how `$i *= 2` is more "suspicious" than `$i++`.

**Comment by kalessil** [comment]
Thank you for reporting @zerkms. That's false-positive, I'll take care of it.

**Comment by kalessil** [comment]
Fixed
